### DISCUSSION TITLE
Search nodes auto rebalance ?

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#10980] [2.4] [answered]
Does searched nodes get rebalanced after one have been offline for a while or a new node is added?

**Comment by jaypfe** [comment, marked as answer]
They should load balance as referenced in this diagram
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/107776997/d370b7e4-40a3-4aeb-a6ab-1434c5eb0ecf)

https://docs.securityonion.net/en/2.4/architecture.html

However, I have not personally been witness to this as I only operate one search node.
